src/test/java/org/simplify4u/plugins/MavenCompilerUtilsTest.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2017 Danny van Heumen
```

Comment:
put current year

Reply:
I feel like an idiot for making these kinds of trivial mistakes :-/ Thanks for catching this!